GDEdit Web/GDE.Web/Pages/Counter.razor

```diff
@@ -0,0 +1,16 @@
+@page "/counter"
+
+<h1>Counter</h1>
+
+<p>Current count: @currentCount</p>
+
+<button class="btn btn-primary" @onclick="@IncrementCount">Click me</button>
+
+@code {
```

Comment:
Can we customize where the fuck the brackets are being placed?

Reply:
yes we can.